addons/project/models/project.py

```diff
@@ -705,6 +705,11 @@ def _compute_subtask_count(self):
         for task in self:
             task.subtask_count = len(self._get_all_subtasks())
 
+    @api.onchange('company_id')
+    def _onchange_task_company(self):
+        if self.project_id.company_id != self.company_id:
+            self.project_id = False
```

Comment:
Coucou new onchange in a task aimed at removing onchanges :laughing:

Reply:
As explained in the review of the task #2169100, replacing this onchange by a compute field cause an infinite recursive loop on triggers. I think this onchange is part of a small list of onchange that need more thinking before removing them.

Reply:
Alright :)